doc/input-output-reference/src/overview/group-plant-equipment.tex

```diff
@@ -4436,12 +4436,14 @@ \subsection{Water to Water Heat Pumps}\label{water-to-water-heat-pumps}
 
 This model is a parameter estimation based model that uses physical parameters generated from the catalog data. The physical parameters are then used to predict the performance of the heat pump using thermodynamic laws and heat transfer equations. The heat pump has a reciprocating compressor that serves both the hot water and the chilled water loop. Note that this model is currently ``hardwired'' to fluid property R22. FluidProperties:* objects for R-22 must be included in the idf file. These may be copied from DataSets\\FluidPropertiesRefData.idf.
 
-Descriptions and strength of each respective model is available in the following references:
+Descriptions and strengths of these two models are available in the following references:
 
 Jin, Hui. 2002. Parameter Estimation Based Models of Water Source Heat Pumps. Phd. Thesis, Department of Mechanical and Aerospace Engineering, Oklahoma State University. (downloadable from \href{http://www.hvac.okstate.edu}{http://www.hvac.okstate.edu/})
 
 Tang,C. C. 2005. Modeling Packaged Heat Pumps in Quasi-Steady State Energy Simulation Program. M.S. Thesis. Department of Mechanical and Aerospace Engineering, Oklahoma State University. (downloadable from \href{http://www.hvac.okstate.edu}{http://www.hvac.okstate.edu/})
 
+Beginning with version 9.1, an EIR-formulated water-to-water heat pump model was added.  This formulation allows the user to enter curves for specifying the capacity and energy usage in a more generic way, allowing the curves to be applicable to a wide variety of components.  With the original equation fit model, the curve form was locked in to a specific set of linear coefficients.  With the new form, the curve can be a quadratic function of temperatures, or an even more generic lookup table.
```

Comment:
This gets lost along the way - maybe add the object names in bold like the others have above?

Reply:
Added some bold headings to help make it stand out.